airbyte-integrations/connectors/source-sftp-bulk/metadata.yaml

```diff
@@ -1,14 +1,14 @@
 data:
   resourceRequirements:
     jobSpecific:
-    - jobType: check_connection
```

Comment:
This has been updated because some metadata changes have been pushed on master without formatting. I didn't really changed source-sftp-bulk just for fun